porcupine/plugins/run/dialog.py

```diff
@@ -105,8 +105,14 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
             validated_callback=self.update_run_button,
         )
 
-        sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_text = "\n".join("\t•  {%s} = %s" % pair for pair in substitutions.items())
+        sub_textbox = textutils.create_passive_text_widget(
+            content_frame, height=len(substitutions) + 2, tabs=10
```

Comment:
This makes the dialog a bit too wide to my taste. It looks better when the width is determined from the radio button texts.
```suggestion
            content_frame, width=1, height=len(substitutions) + 2, tabs=10
```